pkgs/servers/web-apps/wallabag/default.nix

```diff
@@ -1,41 +1,50 @@
-{ lib, stdenv, fetchurl }:
-
-stdenv.mkDerivation rec {
+{ lib
+, stdenv
+, fetchurl
+}:
+
+# Point the environment variable $WALLABAG_DATA to a data directory
+# that contains the folder `app/config` which must be a clone of
+# wallabag's configuration files with your customized `parameters.yml`.
+# These need to be updated every package upgrade.
+#
+# After a package upgrade, empty the `var/cache` folder or unexpected
+# error will occur.
+
+let
   pname = "wallabag";
-  version = "2.4.2";
-
-  # remember to rm -r var/cache/* after a rebuild or unexpected errors will occur
+  version = "2.4.3";
+in
```

Comment:
Why are we now switching things over to let in from rec? The performance improvement is negliable and in the bytes.

Reply:
This would be a prime use case for #119942, but it isn't merged yet.

Reply:
No style guide, I prefer this style, you prefer the other one. :shrug: